### PR TITLE
[CI/CD] Automate Cocoapods deployments in GitHub actions

### PR DESCRIPTION
**Pull request by muryno**

### CODE REVIEW

**Comment by omar-salah-elboredy** [comment]
@muryno where exactly is the deployment to Cocoapods.org step?
I believe that you are only one step away from the deployment, you've done the build and test phases.
You can use this example to help https://github.com/michaelhenry/deploy-to-cocoapods-github-action#example

**Review by omar-salah-elboredy — CHANGES_REQUESTED** [review]
I think we need to add deployment step

**Comment by muryno** [comment]
> @muryno where exactly is the deployment to Cocoapods.org step? I believe that you are only one step away from the deployment, you've done the build and test phases. You can use this example to help https://github.com/michaelhenry/deploy-to-cocoapods-github-action#example
Screenshot 2022-08-13 at 10 04 34


github action have the plugin which I added

File: .github/workflows/ios.yml [hunk 1]
```diff
@@ -0,0 +1,50 @@
+name: Deploy to Cocoapod Action
+  # You may pin to the exact commit or the version.
+  # uses: michaelhenry/deploy-to-cocoapods-github-action@745686ab065f90596e0d5cfcf97bb2416d94262e
+  uses: michaelhenry/deploy-to-cocoapods-github-action@1.0.10
+  with:
+    # The Cocoapods Trunk Token
+    COCOAPODS_TRUNK_TOKEN:
```
Comment (thomasvidas) [inline .github/workflows/ios.yml:7]: I'll add the trunk token when we're ready to actually auto-deploy the SDK

**Review by thomasvidas — APPROVED** [review]

**Review by omar-salah-elboredy — APPROVED** [review]

**Pull request merged**